kouncil-frontend/pom.xml

```diff
@@ -60,15 +60,15 @@
             </configuration>
           </execution>
 
-          <execution>
-            <id>yarn run test</id>
-            <goals>
-              <goal>yarn</goal>
-            </goals>
-            <configuration>
-              <arguments>test</arguments>
-            </configuration>
-          </execution>
+          <!--          <execution>-->
```

Comment:
To tak ma zostać zakomentowane?

Reply:
niestety karma zwraca zly exit code jak jest 0 testow

Reply:
Jak dopiszemy jakiś sensowny test to można będzie włączyć